### DISCUSSION TITLE
Thanks GgorAa

### DISCUSSION THREAD

**Discussion by iialghamrii** [GGorAA/Hackintosh-840#12] [Q&A] [answered]
You save for me alot of time 
Your efi work for me very good 
Two thing I miss 
1: my wifi connect not disconnected but it's maybe slowely than when I used windows
2:I can't make dual boot with windows I try all ways in video at youtube for make it in one hard or two hard drive
Never mind for 1 
And I hope to help for number 2
And thanks for your time for do that❤️

**Comment by ReQuasimodo** [comment]
try <a href="https://sourceforge.net/projects/refind/files/0.12.0/">rEFInd</a> for multiboot.

**Comment by voltangle** [comment, marked as answer]
About the WiFi issue: it is not an EFI problem, that's a WiFi kext being in alpha. It is unstable, but can be used. That is the explanation of broken WiFi on this Hackintosh.

About multiboot: just use [rEFInd](https://sourceforge.net/projects/refind/files/0.12.0/), as mentioned earlier in this thread

Also thanks for the nice words :D
